src/opendirectory/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+project(opendirectory_internal)
+
+add_library(opendirectory_internal src/odutils.c src/odconstants.c)
```

Comment:
What is `opendirectory_internal` going to be used for?

Reply:
It is going to be used for CFOpenDirectory. In particular, these files.

- [odxpc.c](https://github.com/darlinghq/darling-opendirectory/blob/c71415e716a8cba3e1539236b49de8abb3de8fe7/Core/odxpc.c#L6)
- [transaction.c](https://github.com/darlinghq/darling-opendirectory/blob/c71415e716a8cba3e1539236b49de8abb3de8fe7/Core/transaction.c#L28)
- [CFODQuery.c](https://github.com/darlinghq/darling-opendirectory/blob/c71415e716a8cba3e1539236b49de8abb3de8fe7/Core/CFODQuery.c#L30)
- [CFODNode.c](https://github.com/darlinghq/darling-opendirectory/blob/c71415e716a8cba3e1539236b49de8abb3de8fe7/Core/CFODNode.c#L31)
- [CFODSession.c](https://github.com/darlinghq/darling-opendirectory/blob/c71415e716a8cba3e1539236b49de8abb3de8fe7/Core/CFODSession.c#L32)
- [CFODRecord.c](https://github.com/darlinghq/darling-opendirectory/blob/c71415e716a8cba3e1539236b49de8abb3de8fe7/Core/CFODRecord.c#L29)